ohtu2021-kvantti/WebMark
# Refactor tests

## Summary
Tests refactored to match the views. Closes #108 . Some implementations of test helped methods were used but most of the files are still polluted by fixated test data.
## How to test
Check if CI passes **and** the tests seem to be in their view-corresponding test files.
